app/assets/stylesheets/basic.scss

```diff
@@ -1 +1,27 @@
+#background {
+  background: url("http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
+              no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  min-height: 100%
+}
 
+#page-content {
+  margin-top: 20px;
+  margin-left: 10px;
+  margin-right: 10px;
+  padding: 30px 20px;
+  background-color: rgba(158, 158, 158, 0.9);
+}
+
+#title-box {
+  padding-top: 70px;
+  padding-bottom: 70px;
+}
+#title-text {
+  color: #F5F5F5;
```

Comment:
Color literals like `#F5F5F5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F5F5F5` should be written as `#f5f5f5`